media-libs/opencv/opencv4-4.4.0.recipe

```diff
@@ -0,0 +1,277 @@
+SUMMARY="Open Source Computer Vision Library"
+DESCRIPTION="A collection of algorithms and sample code for various \
+computer vision problems."
+HOMEPAGE="https://opencv.org/"
+COPYRIGHT="2000-2020, Intel Corporation
+	2009-2011, Willow Garage Inc.
+	2009-2016, NVIDIA Corporation
+	2010-2013, Advanced Micro Devices, Inc.
+	2015-2016, OpenCV Foundation
+	2015-2016, Itseez Inc.
+	2019-2020, Xperience AI"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/opencv/opencv/archive/$portVersion.tar.gz"
+SOURCE_FILENAME="opencv-$portVersion.tar.gz"
+CHECKSUM_SHA256="bb95acd849e458be7f7024d17968568d1ccd2f0681d47fd60d34ffb4b8c52563"
+SOURCE_DIR="opencv-$portVersion"
+PATCHES="opencv-$portVersion.patchset"
+
+ARCHITECTURES="?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+fi
+
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+libVersion="$portVersion"
+libVersionCompat="$libVersion compat >= ${libVersion%.*}"
+
+PROVIDES="
+	opencv4$secondaryArchSuffix = $portVersionCompat
+	lib:libopencv_calib3d$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_core$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_dnn$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_features2d$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_flann$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_gapi$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_highgui$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_imgcodecs$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_imgproc$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_ml$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_objdetect$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_photo$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_stitching$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_video$secondaryArchSuffix = $libVersionCompat
+	lib:libopencv_videoio$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libavcodec$secondaryArchSuffix
+	lib:libavformat$secondaryArchSuffix
+#	lib:libavresample$secondaryArchSuffix
+	lib:libavutil$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgdal$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libgphoto2$secondaryArchSuffix
+	lib:libgstapp_1.0$secondaryArchSuffix
+	lib:libgstreamer_1.0$secondaryArchSuffix
+	lib:libgstvideo_1.0$secondaryArchSuffix
+	lib:libharfbuzz$secondaryArchSuffix
+	lib:libhdf5$secondaryArchSuffix
+#	lib:libilmimf$secondaryArchSuffix #openexr
+	lib:libintl$secondaryArchSuffix
+	lib:libjasper$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:liblapack$secondaryArchSuffix
+	lib:libopenblas$secondaryArchSuffix
```

Comment:
For some reason missing for secondary architecture build (32 bit)?
```diff
+	lib:libopenjp2$secondaryArchSuffix
```
Added them in the recipe (lib + devel section)